config/locales/en.yml

```diff
@@ -4087,6 +4087,8 @@ en:
     destroy:
       success: Webhook endpoint successfully deleted
       error: Webhook endpoint failed to delete
+    test:
+      success: Some test data has been sent to the webhook url
```

Comment:
At this stage, all we know for sure is that it has been enqueued. Maybe we should say "test data _will_ be sent"?